feature-libs/checkout/components/components/delivery-mode/delivery-mode.component.html

```diff
@@ -1,128 +1,65 @@
-<ng-container *cxFeatureLevel="'!4.2.2'">
-  <div [formGroup]="mode">
-    <div class="row">
```

Comment:
Can we consider a different name for `CheckoutDeliveryFacade`'s `getSetDeliveryModeInProcess()`?
E.g. `isSetDeliveryModeBusy()`? Please feel free to suggest another name, if you think it's more suitable.

Reply:
Similarly, can rename the `DeliveryModeComponent#deliveryModeSetInProcess$`  property to match the above whatever name we go with for the method above?